File: rb/Exercise8.rb
print "How many people? "
people = gets.chomp .to_i 
print "How many pizzas do you have? "
pizzas = gets.chomp.to_i 
print "How many slices are in each pizza? "
slices = gets.chomp.to_i

per_person = pizzas*slices/people
leftover = slices%people

puts "#{people} people with #{pizzas} pizzas:"
puts "Each person gets #{per_person} pieces of pizza."
puts "There are #{leftover} leftover pieces."

File: py/Exercise8.py
#Exercise 8: Pizza Party

people = int(input("How many people? "))
pizzas = int(input("How many pizzas do you have? "))
pizza_slices = int(input("How many slices are in each pizza? "))
per_person = pizzas*pizza_slices//people
print(f"{people} people with {pizzas} pizzas")
print(f"Each person gets {per_person} pieces of pizza.")
print(f"There are {pizza_slices%people} leftover pieces.")
